README.md

```diff
@@ -47,36 +67,42 @@ export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 ```
 
-To speedup the transfer of files when kitchens are run on ec2 instances, the [transport](https://docs.chef.io/workstation/config_yml_kitchen/#transport-settings) selected is `kitchen-transport-speedy` https://github.com/criteo/kitchen-transport-speedy.
-
-To install `kitchen-transport-speedy` in the kitchen embedded ruby environment: `/opt/cinc-workstation/embedded/bin/gem install kitchen-transport-speedy`
+To speed up the transfer of files when kitchens are run on ec2 instances,
+the [transport](https://docs.chef.io/workstation/config_yml_kitchen/#transport-settings) selected is [kitchen-transport-speedy](https://github.com/criteo/kitchen-transport-speedy).
+To install `kitchen-transport-speedy` in the kitchen embedded ruby environment please run: `/opt/cinc-workstation/embedded/bin/gem install kitchen-transport-speedy`.
 
-In order to test on docker containers, you also need docker installed on your environment.
+In order to test on docker containers, you also need `docker` installed on your environment.
+Please note that not all the tests can run on docker so in any case we need to validate our recipes on EC2.
 
-### Helpers
+### Kitchen tests helpers
 
 `kitchen.docker.sh` and `kitchen.ec2.sh` help you run kitchen tests virtually without any further environment setup.
+They are wrappers of the `kitchen` command, so you can pass them all the options exposed by it. See `kitchen --help` for more details. 
 
-You must however do some initial setup on your AWS account in order to be able to use defaults from `kitchen.ec2.sh`.
+You must do some initial setup on your AWS account in order to be able to use defaults from `kitchen.ec2.sh`.
 Take a look at comments at the top of the script in order to understand how to use it.
-
 Both scripts can be run as follows:
 
 ```
-kitchen.*.sh <context> <kitchen parameters>
+kitchen.<driver>.sh <context> <kitchen parameters>
 ```
 
-`<context>` is your test context, like `recipes-config` or `resources-install`.
+`<context>` is your test context, like `environment-config` or `platform-install`.
+For example `./kitchen.docker.sh platform-install ...` will run kitchen tests in `cookbooks/aws-parallelcluster-platform` directory on docker.
 
 
-For instance:
-
+Other examples of submission are:
 ```
-./kitchen.docker.sh recipes-install test cfnconfig-mixed -c 5 -l debug
+# Run supervisord kitchen test from file kitchen.platform-install.yml in cookbooks/aws-parallelcluster-platform directory,
+# for all OSes (concurrency 5) and log level debug
+./kitchen.docker.sh platform-install test supervisord -c 5 -l debug
 
-./kitchen.ec2.sh resources-config test -c 5
+# Run converge phase only of kitchen from file kitchen.environment-config.yml in cookbooks/aws-parallelcluster-environment directory,
```

Comment:
Would it be worth explaining when we might need to run `converge` or `verify` specifically? It might be trivial after a while, but might not be straightforward at the beginning.

Reply:
done.